### ISSUE TITLE
[BUG] core-daemon fails to start on 5.3

### ISSUE THREAD

**Issue by GarettFox** [coreemu/core#264]
I recently updated from 4.8 to 5.3, and ever since, I have not been able to run the core-daemon consistently.

**To Reproduce**
1. run `/etc/init.d/core-daemon start`
2. run `/etc/init.d/core-daemon status`

Status returns `core-daemon is stopped`. Sometimes works again upon reinstalling core entirely.

**Expected behavior**
I expect tthe core-daemon to remain running when I start it more than once.

**Screenshots**
The last few lines of core-daemon.log are:

`
2019-07-02 03:12:26,328 - DEBUG - coreservices:add_services - not loading service: service(xorp_rtrmgr) missing executable: xorp_rtrmgr
2019-07-02 03:12:26,328 - DEBUG - coreservices:add - loading service: class(EmaneTransportService) name(transportd)
2019-07-02 03:12:26,329 - DEBUG - coreemu:load_services - custom service paths: None
2019-07-02 03:12:26,329 - ERROR - core-daemon:cored - error starting main server on:  0.0.0.0:4038
Traceback (most recent call last):
  File "/usr/local/bin/core-daemon", line 68, in cored
    server = CoreServer(address, CoreHandler, cfg)
  File "/usr/lib/python3/dist-packages/core/api/tlv/coreserver.py", line 29, in __init__
    socketserver.TCPServer.__init__(self, server_address, handler_class)
  File "/usr/lib/python3.5/socketserver.py", line 440, in __init__
    self.server_bind()
  File "/usr/lib/python3.5/socketserver.py", line 454, in server_bind
    self.socket.bind(self.server_address)
OSError: [Errno 98] Address already in use
2019-07-02 03:12:26,329 - INFO - coreemu:shutdown - shutting down all sessions
`

**Desktop (please complete the following information):**
 - OS: Debian GNU/Linux 9 (stretch) and Kali GNU/Linux Rolling
 - CORE Version 5.3
 - EMANE Version 1.2.3

**Additional context**
Not sure if this is relevant, but installing via the amd64 .deb packages doesn't seem to include the /etc/core/*.conf files so I've had to manually replace those each time I reinstall.

**Comment by emirljuca** [comment]
Not sure if you have tried this, but did you uninstall 4.8 before installing 5.3?

**Comment by bharnden** [comment]
The error you are presenting is indicating that something is already running and service on the same address. Potentially an old version of CORE. Ideally you would have cleared out anything related to that first, before trying to update.

You may also have older legacy code stuck under /usr/local path as suggested by the /usr/local/bin/core-daemon line.

**Comment by bharnden** [comment]
I also just checked installing both current deb packages on the release page and they installed file to /etc/core as well.

**Comment by GarettFox** [comment]
I didn't uninstall 4.8 the first time, but I have since rm -rf'd everything listed [here](http://coreemu.github.io/core/install.html#installed-files) multiple times to no avail. I did have a lot of core-related files in `/usr/local` which I have deleted. I also apparently had config files for core-daemon present so I ran `dpkg --purge core-daemon`. I tried `dpkg --purge core-gui` too but there was nothing to purge for that. I also ran `apt-get purge core` and then reinstalled the package and now it's installed files to `/etc/core` without intervention. After reinstalling all of the python dependencies as well, core-daemon can restart successfully, so it looks like I just had a lot of catastrophic outdated configs and data.
